ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/PruningResult.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v1
+
+import io.grpc.Status
+
+sealed trait PruningResult extends Product with Serializable
+
+/**
+  * Pruning has been performed. Indicates readiness to prune the ledger api server at the same offset passed to the
+  * WriteService.
```

Comment:
> Indicates readiness to prune the ledger api server

What happens if the ledger API server does not prune its index database at the same offset?

Reply:
Let's sync and see if the blurb I am adding to the interface addresses your question or if there is another nuance I may be missing.